src/rest.rs

```diff
@@ -1005,6 +1005,32 @@ fn handle_request(
 
             json_response(tx.remove(0), ttl)
         }
+        (&Method::POST, Some(&"txs"), None, None, None, None) => {
```

Comment:
Is this only going to be exposed to the backend?

If so, we should have the first path be the same for all internal endpoints...

Reply:
`/mempool` is fine, then we can just whitelist the backend to access `/mempool/*`

Whatever it is, endpoints we don't want to expose need to be protected somehow.

`protected` or `internal` might be  better (but require changes on the backend as well)

Reply:
I think we use `/mempool/recent` in the frontend, so we can't block `/mempool/*` completely.

But a common prefix for all of the non-public endpoints seems like a good idea.

Reply:
Yeah, let's grow the scope of this PR (and the corresponding backend PR) to unify all the endpoints we want to block publicly under one root.

`internal` seems fine.

Maybe we should feature gate them, and update the production deployment instructions etc. to use that feature flag... that way if someone wants to use ONLY electrs, they don't default to exposing the internal endpoints.